src/plugins/telemetry_management_section/public/plugin.tsx

```diff
@@ -25,23 +25,30 @@ export interface TelemetryManagementSectionPluginDepsSetup {
 
 export class TelemetryManagementSectionPlugin {
   public setup(
-    core: CoreSetup,
+    core: CoreSetup<{ docLinks: DocLinksStart }>,
```

Comment:
I don't think you need to specify the core start dependencies. AFAIK, this is to specify the PluginDepsStart contracts only.
```suggestion
    core: CoreSetup,
```

Reply:
Good catch!